tools/hive-bigquery/bigquery_component.py

```diff
@@ -0,0 +1,725 @@
+                        query = "UPDATE %s SET bq_job_status='DONE' WHERE " \
+                                "bq_job_id='%s'" % (
+                                    hive_table_model.tracking_table_name,
+                                    bq_job_id)
+                        mysql_component.execute_transaction(query)
+                        print_and_log(
+                            "Updated BigQuery load job {} status RUNNING --> "
+                            "DONE".format(
+                                bq_job_id))
+                        # Deletes the data file in GCS
+                        gcs_component.delete_file(gcs_bucket_name,
+                                                  gcs_file_path)
+                    # Job finished with error
+                    elif job.errors is not None:
+
+                        if bq_job_retries == bq_load_job_max_retries:
+                            query = "UPDATE %s SET bq_job_status='FAILED' " \
+                                    "WHERE bq_job_id='%s'" % (
+                                        hive_table_model.tracking_table_name,
+                                        bq_job_id)
+                            mysql_component.execute_transaction(query)
+                            print_and_log(
+                                "BigQuery job {} failed.Tried for a maximum "
+                                "of 3 times.Updated status RUNNING --> "
+                                "FAILED".format(
+                                    bq_job_id))
+                        else:
+                            query = "UPDATE %s SET bq_job_status='TODO'," \
+                                    "bq_job_retries=%d WHERE bq_job_id='%s'" % (
+                                        hive_table_model.tracking_table_name,
+                                        bq_job_retries + 1, bq_job_id)
+                            mysql_component.execute_transaction(query)
+                            print_and_log(
+                                "BigQuery job {} failed.Updated status "
+                                "RUNNING --> TODO & increased retries count "
+                                "by 1".format(
+                                    bq_job_id))
+
+                elif job.state == 'RUNNING':
+                    # Count of jobs which are still in running state
+                    count += 1
+                else:
+                    logger.debug(
+                        "job id %s job state %s", bq_job_id, job.state)
+            if count == 0:
+                print_and_log(
+                    "No BigQuery job is in RUNNING state. No values to update")
+                break
+            print_and_log("Waiting for 1 min..")
+            time.sleep(60)
+            print_and_log(
+                "Fetching information about BigQuery load jobs from tracking "
+                "table...")
+            query = "SELECT gcs_file_path,bq_job_id,bq_job_retries FROM %s " \
```

Comment:
Inconsistent use of String formatting. Let's switch this to new format if we can